src/app.py

```diff
@@ -203,17 +224,91 @@ def get_file_id(file_uuid):
     except Exception as e:
         eMsg = str(e)
         logger.error(e, exc_info=True)
-        return(Response("Unexpected error: " + eMsg, 500))
+        return (Response("Unexpected error: " + eMsg, 500))
+
 
 @app.route('/<uuid>/ancestors')
```

Comment:
Please also add the method to this route:
`@app.route('/<uuid>/ancestors', methods=["GET"])`